apprise/plugins/NotifyGoogleChat.py

```diff
@@ -164,6 +175,13 @@ def __init__(self, workspace, webhook_key, webhook_token, **kwargs):
             self.logger.warning(msg)
             raise TypeError(msg)
 
+        # Thread Key (associated with thread)
+        try:
+            self.threadkey = validate_regex(threadkey)
+        except:
+            msg = 'No valid threadKey found in provided URI.'
```

Comment:
The code will produce a warning message for everyone; is this what you expect?

Perhaps:
```python
    if threadkey:
       # Contain check/test only if the threadkey was provided:
        try:
            self.threadkey = validate_regex(threadkey)
        except:
            msg = 'No valid threadKey found in provided URI.'
```

Reply:
It is may be irrelevant ... If you provide a threadKey and still don't get your message in the same thread, you may be more looking in .DEBUG or .INFO lines ... I agree.

Reply:
fixed in .debug rather than .warning

Reply:
fixed to ensure a 100% codecov coverage